### PR TITLE
feat: Integrate demo store credential with edge-service

### LINKED ISSUE #50: Integrate demo store credential with edge-service

### PR DESCRIPTION
**Pull request by talwinder50**
closes #50

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: pkg/restapi/issuer/operation/operations_test.go [hunk 1]
```diff
@@ -140,33 +168,79 @@ func TestOperation_Callback_CreateCredential_Error(t *testing.T) {
...
+		fmt.Fprintln(w, "{}")
+	}))
+	defer cms.Close()
+
+	vcs := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusCreated)
+		fmt.Fprintln(w, "{}")
+	}))
+
+	defer cms.Close()
```
Comment (sandrask) [inline pkg/restapi/issuer/operation/operations_test.go]: vcs.Close()

File: pkg/restapi/issuer/operation/operations_test.go [hunk 2]
```diff
@@ -140,33 +168,79 @@ func TestOperation_Callback_CreateCredential_Error(t *testing.T) {
...
+
+	data, status, err := handleRequest(handler, headers, callback)
+	require.NoError(t, err)
+	require.Equal(t, http.StatusInternalServerError, status)
+	require.Contains(t, data.String(), "failed to store credential")
+}
+
 func TestOperation_CreateCredential_Error(t *testing.T) {
-	svc := New(&Config{TokenIssuer: &mockTokenIssuer{}, TokenResolver: &mockTokenResolver{}})
+	t.Run("create credential error unexpected end of JSON input ", func(t *testing.T) {
```
Comment (sandrask) [inline pkg/restapi/issuer/operation/operations_test.go]: Can you revert this change?

File: pkg/restapi/issuer/operation/operations_test.go (hunk 1 above)
Comment (talwinder50) [reply pkg/restapi/issuer/operation/operations_test.go]: done

File: pkg/restapi/issuer/operation/operations_test.go (hunk 2 above)
Comment (talwinder50) [reply pkg/restapi/issuer/operation/operations_test.go]: done
